README.md

```diff
@@ -33,6 +33,9 @@ This includes a default role (`/CQSE/TEAMSCALE_RFC`) for RFC access from Teamsca
 
 **Optional, alternatively an appropriate role can be defined manually, see [section *Authorizations for SAP Teamscale User* in the Teamscale documentation](https://docs.teamscale.com/tutorial/sap-integration/#authorizations-for-sap-teamscale-user).**
 
+**CQIK900945** (files deletion/K900945.CQI / deletion/R900945.CQI)  
+This is the removal transport for the Teamscale Connector for SAP Netweaver ABAP. It removes all objects in the `/CQSE/` namespace and its subpackages.
```

Comment:
Also: What do you think of moving this down to a new "Removal" (or "Deletion") section below "Installation". Would be a little more separated, and since the files are in their own directory, might even make more sense.